package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "node-firefox-forward-ports",
+  "version": "1.0.0",
+  "description": "Find attached FirefoxOS devices using ADB",
+  "main": "index.js",
+  "dependencies": {
+    "adbkit": "^2.1.6",
+    "es6-promise": "^2.0.1",
+    "portfinder": "^0.4.0"
+  },
+  "devDependencies": {
+    "gulp": "^3.8.10",
+    "gulp-jscs": "^1.3.1",
```

Comment:
I believe this is the case, and because `build-tools` uses `gulp` as an actual dependency you don't need to list it here. I think.